include/seqan3/alignment/all.hpp

```diff
@@ -10,20 +10,295 @@
+ * helpful if the region in which the optimal alignment exists is known a priori. To specify the banded alignment
+ * the developer can use the seqan3::align_cfg::band option initialised with a seqan3::static_band.
+ * The seqan3::static_band is constructed with a lower bound and an upper bound. The upper bound must always be greater
+ * than or equal to the lower bound. To position the band, imagine a rectangle where the first sequence is written on
+ * top and the second sequence along the left vertical side. A negative bound implies a start of the band within the
+ * vertical part while a positive bound implies a start within the top part of this rectangle at the respective
+ * position.
+ *
+ * ## Global and local alignments
+ *
+ * The standard global and local alignments can be configured using the configuration seqan3::align_cfg::mode.
+ * The global alignment can be further refined using the seqan3::align_cfg::sequence_ends configuration.
+ * This configuration allows to enable, respectively disable the scoring of leading and trailing gaps at the respective
+ * sequence ends. This option is not available for the local alignment where scoring gaps at the ends of the sequences
+ * is always disabled.
+ *
+ * ## Computing edit distance
+ *
+ * A special implementation of the global sequence alignment is the edit distance. This algorithm computes merely
+ * the number of edit operations (substitutions, insertions and deletions). It can only be computed with a scoring
+ * scheme initialised with `0` for matching letters and `-1` for mismatching letters and a gap scheme with `-1` for
+ * inserting or deleting a gap. Currently, it is only available for nucleotide scoring schemes. There is a special
+ * edit distance shortcut config seqan3::align_cfg::edit which is initialised with the correct scoring and gap scheme.
+ *
+ * If the alignment configurations allows the computation of the edit distance it will be preferred over the
+ * standard pairwise alignment since it is much faster.
+ * The edit distance algorithm can be further specialised with the seqan3::align_cfg::max_error configuration. This
+ * limits the maximal number of errors allowed and can further increase the performance of the algorithm.
+ *
+ * \note If there was a configuration that is not suitable for the edit distance algorithm the standard alignment
+ * algorithm is executed as a fallback.
+ *
+ * # Parallel alignment execution
+ *
+ * SeqAn's alignment algorithm is internally accelerated using multi-threading. To compute the alignments in parallel
+ * one can call seqan3::align_pairwise with a seqan3::par execution policy.
+ * This will automatically spawn std::thread::hardware_concurrency many threads that will compute the alignments
+ * concurrently. By default the algorithm is called with the seqan3::seq execution policy, which will trigger
+ * sequential execution of the alignments.
  */
+
```

Comment:
I really like this page, but it is very sparse on snippets. Does the documentation for the align_pairwise function have more snippets. What parts are documented there and what parts are documented here?

Reply:
later